### PR TITLE
magento/devdocs#: Add information about required Apache modules

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds a new section into [Apache](https://devdocs.magento.com/guides/v2.3/install-gde/prereq/apache.html) page about required Apache modules

 mentioned by @ihor-sviziev  in https://github.com/magento/devdocs/issues/6689:

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/install-gde/prereq/apache.html

## Links to Magento source code

*  [mod_deflate.c](https://github.com/magento/magento2/blob/2.3.4/pub/.htaccess#L87)
*  [mod_expires.c](https://github.com/magento/magento2/blob/2.3.4/pub/.htaccess#L172)
*  [mod_headers.c](https://github.com/magento/magento2/blob/2.3.4/pub/.htaccess#L229)
*  [mod_php7.c](https://github.com/magento/magento2/blob/2.3.4/pub/.htaccess#L54)
*  [mod_rewrite.c](https://github.com/magento/magento2/blob/2.3.4/pub/.htaccess#L124)
*  [mod_security.c](https://github.com/magento/magento2/blob/2.3.4/pub/.htaccess#L79)
*  [mod_ssl.c](https://github.com/magento/magento2/blob/2.3.4/pub/.htaccess#L115)

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/install-gde/prereq/apache.md [hunk 1]
```diff
@@ -37,6 +37,18 @@ Magento uses server rewrites and `.htaccess` to provide directory-level instruct
...
 Failure to enable these settings typically results in no styles displaying on your storefront or Admin.
 
+## Apache required modules {#apache-required-modules}
+
+Magento requires the following Apache modules be installed:
+
+*  [mod_deflate.c](https://httpd.apache.org/docs/2.4/mod/mod_deflate.html)
+*  [mod_expires.c](https://httpd.apache.org/docs/2.4/mod/mod_expires.html)
+*  [mod_headers.c](https://httpd.apache.org/docs/2.4/mod/mod_headers.html)
+*  [mod_php7.c](https://www.php.net/manual/en/security.apache.php)
```
Comment (hostep) [inline src/guides/v2.3/install-gde/prereq/apache.md]: The requirement of mod_php7 is not strictly true. If you are using php-fpm for example this is not necessary. I would remove this line, no?

Comment (atwixfirster) [reply src/guides/v2.3/install-gde/prereq/apache.md]: Hi, @hostep !

Good notice!

May be I should add an `(optional)` word for `mod_php7`.

What do you think? Thanks

Comment (hostep) [reply src/guides/v2.3/install-gde/prereq/apache.md]: Maybe but then I would also change the `Magento requires ...` to `Magento recommends ...` or something like that?

Comment (ihor-sviziev) [reply src/guides/v2.3/install-gde/prereq/apache.md]: @atwixfirster Could you just remove it from the list? It's not actually required, as there few options how to set up apache configuration.

Comment (atwixfirster) [reply src/guides/v2.3/install-gde/prereq/apache.md]: > @atwixfirster Could you just remove it from the list? It's not actually required, as there few options how to set up apache configuration.

@ihor-sviziev ,

 **done**, sir

**Comment by dobooth** [comment]
Thanks, gents!

**Comment by dobooth** [comment]
running tests

**Pull request merged**